### PR TITLE
【doc】update docs for development guide

### PR DESCRIPTION
**Pull request by lilai23**
Related issue

https://github.com/huaweicloud/Sermant/issues/753

What type of PR is this?

doc

Does this PR introduce a user-facing change?

No

Related docs:
(1) Plugin Code Development Guide
[dev_plugin_code.md](https://github.com/lilai23/Sermant/blob/doc_replenish/docs/dev-guide/dev_plugin_code.md)
[dev_plugin_code-zh.md](https://github.com/lilai23/Sermant/blob/doc_replenish/docs/dev-guide/dev_plugin_code-zh.md)
(2) Plugin Module Development Guide
[dev_plugin_module.md](https://github.com/lilai23/Sermant/blob/doc_replenish/docs/dev-guide/dev_plugin_module.md)
[dev_plugin_module-zh.md](https://github.com/lilai23/Sermant/blob/doc_replenish/docs/dev-guide/dev_plugin_module-zh.md)

### CODE REVIEW

File: docs/dev-guide/dev_plugin_code.md [hunk 1]
```diff
@@ -1,340 +1,343 @@
...
   ClassMatcher.nameContains("${class reference array}");
   ```
-  其中`${class reference array}`为被增强类的全限定名可变数组。
+  Where `${class reference array}` is a FQCN mutable array of the enhanced class.
 
-[ClassFuzzyMatcher](../../sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/matcher/ClassFuzzyMatcher.java)（类的模糊匹配器）
+[ClassFuzzyMatcher](../../sermant-agentcore/sermant-agentcore-core/src/main/java/com/huaweicloud/sermant/core/plugin/agent/matcher/ClassFuzzyMatcher.java)（the fuzzy matcher of class name）
 
-- 通过全限定名前缀定位到被增强类，可通过以下方法获取：
+- The enhanced class is located by the prefix of FQCN, which can be obtained by:
```
Comment (justforstudy-A) [inline docs/dev-guide/dev_plugin_code.md]: what meaning of FQCN

File: docs/dev-guide/dev_plugin_code.md [hunk 2]
```diff
@@ -1,340 +1,343 @@
...
 
-由[插件模块开发手册](dev_plugin_module.md)可知，一个`插件主模块(main)`中可能包含以下5种内容：
+## Components
 
-- `插件模块(plugin)`，该模块主要用于声明对宿主应用的增强逻辑
-- `服务模块(service)`，用于为插件包提供服务实现
-- `后端模块(server)`，用于接收插件数据的服务端
-- `前端模块(webapp)`，用于对服务端数据作前端展示
-- `其他模块(other)`，特殊附加件，一般用作调试
+According to the [Plugin Module Development Guide](dev_plugin_module.md), a `main` plugin module contains the following five things:
```
Comment (justforstudy-A) [inline docs/dev-guide/dev_plugin_code.md:24]: The link of Plugin Module Development Guide   is different to it's zh version which relate to title "Add Plugin Module"

File: docs/dev-guide/dev_plugin_code.md (hunk 1 above)
Comment (lilai23) [reply docs/dev-guide/dev_plugin_code.md]: FQCN= Full Qualified Class Name

File: docs/dev-guide/dev_plugin_code.md (hunk 2 above)
Comment (lilai23) [reply docs/dev-guide/dev_plugin_code.md:24]: resolved

File: docs/dev-guide/dev_plugin_code.md (hunk 1 above)
Comment (lilai23) [reply docs/dev-guide/dev_plugin_code.md]: modified to Full Qualified Class Name

**Review by xuezechao1 — DISMISSED** [review]

**Review by provenceee — DISMISSED** [review]

**Review by justforstudy-A — DISMISSED** [review]

**Review by robotLJW — DISMISSED** [review]
/lgtm

**Review by luanwenfei-venus — APPROVED** [review]

**Review by robotLJW — APPROVED** [review]

**Pull request merged**